librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -51,4 +51,31 @@ commands:
             summary: Step until pc == <flag> matching name
             args:
             - name: flag
-              type: RZ_CMD_ARG_TYPE_FLAG
\ No newline at end of file
+              type: RZ_CMD_ARG_TYPE_FLAG
+  - name: dts
+    summary: Debug trace session commands
+    subcommands:
+    - name: dts+
+      summary: Start trace session
+      cname: cmd_debug_start_trace_session
+      args: []
+    - name: dts-
+      summary: Stop trace session
+      cname: cmd_debug_stop_trace_session
+      args: []
+    - name: dtst
+      cname: cmd_debug_save_trace_session
+      summary: Save trace sessions to disk
+      args:
+        - name: dir
+          type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
I think this and all other dir args should be FILE type, so you can benefit from autocompletion... If we want to be even more precise we should probably define a new DIR type

Reply:
Yeah, I thought about that, but it looks like only these two commands have a directory as an argument.
```
[0x00000000]> ?*~dir
| dtst <dir>           # Save trace sessions to disk
| dtsf <dir>           # Load trace sessions to disk
| ls [<arg0> [<arg1> ...]] # List files and directories
| m                    # Make directories and move files
[0x00000000]>
```
So, I guess, we can go with `FILE` then?